site/weavedns.md

```diff
@@ -32,6 +43,12 @@ $ docker attach $shell1
 ...
 ```
 
+Note that it is permissible to register multiple containers with the
+same name. weaveDNS picks one address to return when asked for the
+name. Since weaveDNS removes any container that dies, this is a simple
```

Comment:
Nobody knows how it's meant to be spelled. I did not introduce this inconsistency and am not going to attempt to resolve it in this PR.